RemoteContainers.md

```diff
@@ -0,0 +1,185 @@
+# Using m365pnp/spfx Image in Visual Studio Remote Containers
```

Comment:
Let's update the title to `development container`

Reply:
Done